src/Fixer/PhpUnit/PhpUnitStaticAssertionCallsFixer.php

```diff
@@ -0,0 +1,351 @@
+        'assertNotInstanceOf' => true,
+        'assertNotInternalType' => true,
+        'assertNotIsReadable' => true,
+        'assertNotIsWritable' => true,
+        'assertNotNull' => true,
+        'assertNotRegExp' => true,
+        'assertNotSame' => true,
+        'assertNotSameSize' => true,
+        'assertNotTrue' => true,
+        'assertNull' => true,
+        'assertObjectHasAttribute' => true,
+        'assertObjectNotHasAttribute' => true,
+        'assertRegExp' => true,
+        'assertSame' => true,
+        'assertSameSize' => true,
+        'assertStringEndsNotWith' => true,
+        'assertStringEndsWith' => true,
+        'assertStringEqualsFile' => true,
+        'assertStringMatchesFormat' => true,
+        'assertStringMatchesFormatFile' => true,
+        'assertStringNotEqualsFile' => true,
+        'assertStringNotMatchesFormat' => true,
+        'assertStringNotMatchesFormatFile' => true,
+        'assertStringStartsNotWith' => true,
+        'assertStringStartsWith' => true,
+        'assertThat' => true,
+        'assertTrue' => true,
+        'assertXmlFileEqualsXmlFile' => true,
+        'assertXmlFileNotEqualsXmlFile' => true,
+        'assertXmlStringEqualsXmlFile' => true,
+        'assertXmlStringEqualsXmlString' => true,
+        'assertXmlStringNotEqualsXmlFile' => true,
+        'assertXmlStringNotEqualsXmlString' => true,
+        'attribute' => true,
+        'attributeEqualTo' => true,
+        'callback' => true,
+        'classHasAttribute' => true,
+        'classHasStaticAttribute' => true,
+        'contains' => true,
+        'containsOnly' => true,
+        'containsOnlyInstancesOf' => true,
+        'countOf' => true,
+        'directoryExists' => true,
+        'equalTo' => true,
+        'fail' => true,
+        'fileExists' => true,
+        'getCount' => true,
+        'getObjectAttribute' => true,
+        'getStaticAttribute' => true,
+        'greaterThan' => true,
+        'greaterThanOrEqual' => true,
+        'identicalTo' => true,
+        'isEmpty' => true,
+        'isFalse' => true,
+        'isFinite' => true,
+        'isInfinite' => true,
+        'isInstanceOf' => true,
+        'isJson' => true,
+        'isNan' => true,
+        'isNull' => true,
+        'isReadable' => true,
+        'isTrue' => true,
+        'isType' => true,
+        'isWritable' => true,
+        'lessThan' => true,
+        'lessThanOrEqual' => true,
+        'logicalAnd' => true,
+        'logicalNot' => true,
+        'logicalOr' => true,
+        'logicalXor' => true,
+        'markTestIncomplete' => true,
+        'markTestSkipped' => true,
+        'matches' => true,
+        'matchesRegularExpression' => true,
+        'objectHasAttribute' => true,
+        'readAttribute' => true,
+        'resetCount' => true,
+        'stringContains' => true,
+        'stringEndsWith' => true,
+        'stringStartsWith' => true,
+
+        // TestCase methods
+        'any' => true,
```

Comment:
from which version of phpunit you collected the methods ?
how did you generate it? maybe we could reuse it later, to add method of newer phpunit when it will be released ?

maintaining this list manually will be problematic...